scripts/docker/buildpack-deps/Dockerfile.ubuntu1604.clang.ossfuzz

```diff
@@ -104,11 +104,11 @@ RUN set -ex; \
 # HERA
 RUN set -ex; \
 	cd /usr/src; \
-	git clone --branch="v0.3.0" --recurse-submodules https://github.com/ewasm/hera.git; \
+	git clone --branch="v0.3.1" --recurse-submodules https://github.com/ewasm/hera.git; \
 	cd hera; \
 	mkdir build; \
 	cd build; \
-	cmake -G Ninja -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX="/usr" ..; \
+	cmake -G Ninja -DHERA_DEBUGGING=OFF -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX="/usr" ..; \
```

Comment:
@axic Is there no way to switch this on and off a runtime? I think it would be most useful as an option you could pass to `soltest`/`isoltest` to get extra output when you need it (and when you know you're running just a few tests so you don't get flooded with output).

Reply:
No, it is compile time only. There was a big PR to implement it runtime, but it was abandoned. At the time we were more interested in speed than debuggability.

Reply:
In that case I think it's better to have it disabled by default. I could be very useful when debugging locally but in CI where we'll get debug output for all tests in the repo in one big lump :)